{
  "Severity": "HIGH",
  "UpdatedAt": "2019-02-07T18:01:49Z",
  "Package": {
    "Ecosystem": "PIP",
    "Name": "colander"
  },
  "Advisory": {
    "DatabaseId": 1105,
    "Id": "MDE2OlNlY3VyaXR5QWR2aXNvcnlHSFNBLXJ2OTUtNHd4ai02ZnFx",
    "GhsaId": "GHSA-rv95-4wxj-6fqq",
    "References": [
      {
        "Url": "https://nvd.nist.gov/vuln/detail/CVE-2017-18361"
      },
      {
        "Url": "https://github.com/Pylons/colander/issues/290"
      },
      {
        "Url": "https://github.com/Pylons/colander/pull/323"
      },
      {
        "Url": "https://github.com/Pylons/colander/commit/98805557c10ab5ff3016ed09aa2d48c49b9df40b"
      },
      {
        "Url": "https://github.com/advisories/GHSA-rv95-4wxj-6fqq"
      }
    ],
    "Identifiers": [
      {
        "Type": "GHSA",
        "Value": "GHSA-rv95-4wxj-6fqq"
      },
      {
        "Type": "CVE",
        "Value": "CVE-2017-18361"
      }
    ],
    "Description": "In Pylons Colander through 1.6, the URL validator allows an attacker to potentially cause an infinite loop thereby causing a denial of service via an unclosed parenthesis.",
    "Origin": "UNSPECIFIED",
    "PublishedAt": "2019-02-07T18:18:22Z",
    "Severity": "HIGH",
    "Summary": "Loop with Unreachable Exit Condition ('Infinite Loop') in Pylons Colander",
    "UpdatedAt": "2023-04-20T16:47:23Z",
    "WithdrawnAt": "",
    "CVSS": {
      "Score": 7.5,
      "VectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  },
  "Versions": [
    {
      "FirstPatchedVersion": {
        "Identifier": "1.7.0"
      },
      "VulnerableVersionRange": "\u003c 1.7.0"
    }
  ]
}